kafka/rdkafka/consumer.py

```diff
@@ -20,19 +21,27 @@
 
 class SimpleConsumer(base.BaseSimpleConsumer):
 
-    def __init__(self, client, topic, partitions=None):
-        if isinstance(topic, basestring):
-            topic = client[topic]
-        self._topic = topic
-        self._partitions = partitions or self.topic.partitions # ie all
+    def __init__(self, *args, **kwargs):
```

Comment:
As annoying as it is, duplicating the argspec is a good idea here.  That way code-completion tools will know what's expected. You don't have to reproduce the docstring, but the copypasta for the args will make this much more usable for people.